### PR TITLE
remove deprecated features

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by xabbuh** [comment]
updated to take into account #31975/#32452 too

File: src/Symfony/Component/Stopwatch/Section.php [hunk 1]
```diff
@@ -62,10 +62,6 @@ public function __construct(float $origin = null, bool $morePrecision = false)
      */
     public function get($id)
     {
-        if (null === $id) {
```
Comment (fabpot) [inline src/Symfony/Component/Stopwatch/Section.php]: Can we add the `string` type hint on `$id`?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Stopwatch/Section.php (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Component/Stopwatch/Section.php]: @jschaedl is adding them among others in #32242

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**